suites/pacific/cephadm/tier-0.yaml

```diff
@@ -0,0 +1,122 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/pacific/cephadm/tier-0.yaml
+# Smoke test cases for
+#    - Bootstrap
+#    - Host management
+#    - Ceph role Service deployment,
+#    - Configure client for RGW and RBD systems
+#
+#===============================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      config:
+        is_production: true
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
```

Comment:
Let's not have this key/value in the config. As the primary execution environment is IBM-Cloud and there the DTR is an internal one.

Reply:
@psathyan Could you please elaborate more

Reply:
In IBM environment, the images are pulled from an internal Registry. Ideally, we would like to have overrides mentioned in the suite so that it can be properly handled based on use case.

In this case, `registry-url` becomes an override and it would prevent us from executing in IBM environment. However, we do have a check to switch.

To avoid over dependence on switching in test modules, I prefer test configurations to be environment agnostic hence the request to remove the key/value.